scraping/scraping/start_scripts/zalando_de.py

```diff
@@ -34,34 +34,63 @@ def combine_results(
 
 def male() -> List[dict]:
     path_2_category = {
-        "herrenbekleidung-shirts": "SHIRT",
-        "herrenbekleidung-hemden": "SHIRT",
-        "herrenbekleidung-sweatshirts-hoodies": "SWEATER",
-        "herrenbekleidung-pullover-strickjacken": "SWEATER",
-        "herrenbekleidung-jacken": "JACKET",
-        "herrenbekleidung-maentel": "JACKET",
-        "herrenbekleidung-anzuege": "SUIT",
-        "herrenbekleidung-jeans": "JEANS",
-        "herrenbekleidung-hosen": "PANT",
-        "herrenbekleidung-hosen-shorts": "PANT",
-        "herrenbekleidung-trainingsanzuege-jogger": "TRACKSUIT",
-        "herrenbekleidung-waesche": "UNDERWEAR",
-        "herrenbekleidung-nachtwaesche": "NIGHTWEAR",
-        "herrenbekleidung-bademode": "SWIMMWEAR",
-        # Shoes and Bags
-        "herrenschuhe": "SHOES",
-        "taschen-accessoires-taschen-herren": "BAG",
-        # Sport
-        "sports-herren-shirts": ("SHIRT", {"type": "SPORT"}),
-        "sports-herren-jacken": ("JACKET", {"type": "SPORT"}),
-        "sporthosen-herren": ("PANT", {"type": "SPORT"}),
-        "sports-herren-pullover-sweater": ("SWEATER", {"type": "SPORT"}),
-        "trainingsanzug-herren": ("TRACKSUIT", {"type": "SPORT"}),
-        "funktionsunterwaesche-herren": ("UNDERWEAR", {"type": "SPORT"}),
-        "sports-herren-struempfe": ("UNDERWEAR", {"type": "SPORT"}),
-        "sports-bademode-herren": ("SWIMMWEAR", {"type": "SPORT"}),
-        "sportschuhe-herren": ("SHOES", {"type": "SPORT"}),
-        "sports-taschen-rucksaecke-herren": ("BAG", {"type": "SPORT"}),
+        "/herrenbekleidung-shirts-basic/": ProductCategory.TSHIRT.value,
+        "/herrenbekleidung-shirts-print/": ProductCategory.TSHIRT.value,
+        "/herrenbekleidung-shirts-tops/": ProductCategory.SHIRT.value,
+        "/herrenbekleidung-shirts-poloshirts/": ProductCategory.SHIRT.value,
+        "/herrenbekleidung-shirts-longsleeves/": ProductCategory.TSHIRT.value,
+        "/herrenbekleidung-shirts-sport/": (ProductCategory.TSHIRT.value, {"type": "SPORT"}),
+        "/herrenbekleidung-hemden/": ProductCategory.SHIRT.value,
+        "/herrenbekleidung-sweatshirts-hoodies/": ProductCategory.SWEATER.value,
+        "/herrenbekleidung-pullover-strickjacken/": ProductCategory.SWEATER.value,
```

Comment:
zalando allows to split this category further into:
https://www.zalando.de/damenbekleidung-strickjacken/ which we can assign to `JACKET ` according to GS1 GPC `Brick 10001350 Jacken / Blazer / Strickjacken / Westenand`  and
https://www.zalando.de/damenbekleidung-pullover-sweater-strickpullover/ which we can assign to `SWEATER`